samples/display/cfb_shell/README.rst

```diff
@@ -0,0 +1,47 @@
+.. _cfb_shell_sample:
+
+Character Framebuffer Shell Module Sample
+#########################################
+
+Overview
+********
+A simple shell module that exercises displays using the Character Framebuffer
+subsystem.
+
+Building and Running
+********************
+
+Build the sample app by choosing the OVERLAY_CONFIG for the target board. E.g.:
+
+.. code-block:: console
+
+         cmake -DBOARD=reel_board -DOVERLAY_CONFIG="overlay-reel_board_ssd1673.conf"
+
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/display/cfb_shell
+   :host-os: unix
+   :board:
+   :goals: run
+   :compact:
+
+
+Shell Module Command Help
+=========================
+
+.. code-block:: console
```

Comment:
While this shows the command syntax, it doesn't explain what the (sub)commands actually do.

Reply:
Aren't the command names self-explanatory?

Reply:
I agree with @dbkinder here, anything is better than "[none]" and even for those with help, it is not really easy for someone not familiar with the feature to do anything simple.